apis/v1alpha2/httproute_types.go

```diff
@@ -222,6 +222,7 @@ type HTTPRouteRule struct {
 	// recommended to return a 503 status code.
 	//
 	// Support: Core for Kubernetes Service
+	//
```

Comment:
I took it that the reason for this was that when `<gateway:experimental>` would show up in the docs this would end up in it's own line too, which seems to make sense to me so that it wasn't tailing the `Support: Core` or what have you?